projects/observability/src/shared/components/topology/d3/interactions/drag/topology-node-drag.ts

```diff
@@ -22,7 +24,16 @@ export class TopologyNodeDrag extends TopologyEventBehavior {
     if (topologyData.nodes.length === 0) {
       return EMPTY;
     }
-    const nodeLookup = this.buildLookupMap(topologyData.nodes, node => nodeRenderer.getElementForNode(node));
+
+    const childUserNodesOfGroup = topologyData.nodes
```

Comment:
can we use the groupNode property here as well?

Reply:
Done